grails-app/i18n/messages.properties

```diff
@@ -78,4 +79,10 @@ job.status.IN_PROGRESS=In Progress
 job.status.SUCCEEDED=Succeeded
 job.status.FAILED=Failed
 
+job.reason.NONE=
+job.reason.TIMEOUT_EXPIRED=Timeout expired for job
```

Comment:
@kereid Please help me with the labels for those things. Especially those two - `TIMEOUT_EXPIRED` and `TOO_MANY_FILES`.